controllers/keyword.go

```diff
@@ -28,6 +30,8 @@ type UploadFileForm struct {
 
 func (k *KeywordController) applyRoutes(engine *gin.RouterGroup) {
 	engine.GET("/keyword", k.displayKeyword)
+	engine.GET("/keyword/:keyword_id", k.displayKeywordResult)
+	engine.GET("/keyword/:keyword_id/google-html", k.displayKeywordGoogleHTML)
```

Comment:
How about simplifying it to `/keyword/:keyword_id/html` as it would be flexiblee if other search engines are supported.

Reply:
Good catch. Fixed in 01ba0af024234cbefe807cc480b62eceab3eff80